promql/parser/generated_parser.y

```diff
@@ -582,7 +582,11 @@ label_match_list: label_match_list COMMA label_matcher
                 ;
 
 label_matcher   : IDENTIFIER match_op STRING
-                        { $$ = yylex.(*parser).newLabelMatcher($1, $2, $3);  }
+                        { $$ = yylex.(*parser).newLabelMatcher($1, $2, $3); }
+                | string_identifier match_op STRING
```

Comment:
do we need `string_identifier match_op error` rule? or is the parser error useful without it already?

Reply:
would that be like
`{"x"=}` ?

Reply:
ok yes, adding that makes the error a little better